packages/terra-worklist-data-grid/src/utils/WorklistDataGridUtils.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * The Object representation of the row selection column.
+ */
+const ROW_SELECTION_COLUMN = {
+  id: 'WorklistDataGrid-rowSelectionColumn',
+  width: '40px',
```

Comment:
Is this the value specified by UX?  Eric has been discussing the use of "rem" values.

Reply:
No, we took it from the Clinical Data Grid. I will add a comment in the Row Selection Jira for Eric.